_examples/gae.go

```diff
@@ -0,0 +1,43 @@
+package examples
+
+import (
+	"github.com/mattes/httpexpect"
+	"google.golang.org/appengine/aetest"
+	"net/http"
+	"os"
+	"testing"
+)
+
+var GaeInstance aetest.Instance
+
+func TestMain(m *testing.M) {
+	var err error
+	GaeInstance, err = aetest.NewInstance(nil)
+	if err != nil {
+		panic(err)
+	}
+
+	c := m.Run()
+	GaeInstance.Close()
+	os.Exit(c)
+}
+
+func Router() http.Handler {
+	// return your actual handlers
+	return nil
+}
+
+func NewHttpExpect(t *testing.T) *httpexpect.Expect {
```

Comment:
I wonder if this could be a "first class" func, like 

```
func NewGae(t *testing.T, r http.Handler, i aetest.Instance) *httpexpect.Expect
```
Thoughts?